README.md

```diff
@@ -11,3 +11,7 @@
 ## Category 1
 
 * [Placeholder link]()
+
+## How Do I connect to the Cluster?
+
+* [SSH quickstart](https://linuxhandbook.com/ssh-basics/)
```

Comment:
I like the idea of adding short descriptions to the links, separated by a hyphen. Something like:
```markdown
* [SSH quickstart](link) - Step-by-step ssh and scp guide.
```

For the descriptions, I think we should target keywords that make the list more searchable.

Reply:
sounds good